client/wfprev-war/src/main/angular/src/app/components/search-filter/search-filter.component.html

```diff
@@ -31,7 +31,7 @@
     </div>
 
     <div class="field search-input filter-search-input">
-      <mat-label class="label">Fiscal Year(s)</mat-label>
+      <mat-label class="label">Fiscal Activity</mat-label>
```

Comment:
I dont think we should change this label? this is just year numbers.

Reply:
Reverted to original label